### PR TITLE
Show Profile Image Method

### PR DESCRIPTION
**Pull request by KarimNahas**

### CODE REVIEW

File: app/controllers/image_controller.rb [hunk 1]
```diff
@@ -10,6 +10,19 @@ def index
...
+  def show
+    image = ProfileImage.find_by(id: params[:id])
+    if image
+      if image.cattle.user_id == current_user.id
+        render json: { image: image.fetch_image }, status: :ok
+      else
+        render json: { errors: ['Cattle does not belongs to you'] }, status: :unauthorized
+      end
+    else
+      render json: {}, status: :not_found
```
Comment (FreddieLindsey) [inline app/controllers/image_controller.rb]: No json required

File: config/routes.rb [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
   # Image management
   get    '/cattle/:id/images/', to: 'image#index'
+  get    '/cattle/image/:id',   to: 'image#show'
```
Comment (FreddieLindsey) [inline config/routes.rb]: `/cattle/:cattle_id/image/:id` just to ensure it's correct?

File: spec/controllers/image_controller_spec.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
   before(:all) do
     @user = FactoryGirl.create(:user)
     @cattle = FactoryGirl.create(:cattle, user_id: @user.id)
-    FactoryGirl.create_list(:profile_image, 20, cattle_id: @cattle.id)
+    @image = FactoryGirl.create_list(:profile_image, 20, cattle_id: @cattle.id)[0]
```
Comment (FreddieLindsey) [inline spec/controllers/image_controller_spec.rb:7]: Do we use the list? I'm presuming we do

Comment (KarimNahas) [reply spec/controllers/image_controller_spec.rb:7]: yes we do

File: config/routes.rb (hunk 2 above)
Comment (KarimNahas) [reply config/routes.rb]: doesn't really matter but sure i'll change it

File: app/controllers/image_controller.rb [hunk 4]
```diff
@@ -10,6 +10,19 @@ def index
     end
   end
 
+  def show
+    image = ProfileImage.find_by(id: params[:id])
+    if image
+      if image.cattle.user_id == current_user.id
+        render json: { image: image.fetch_image }, status: :ok
+      else
+        render json: { errors: ['Cattle does not belongs to you'] }, status: :unauthorized
```
[the remark this answers is not in the document]
Comment (KarimNahas) [reply app/controllers/image_controller.rb:21]: It's always nice to have explicit error messages specially that the error reducer can display errors without specifying its origin.

**Pull request merged**